src/fides/api/schemas/messaging/messaging.py

```diff
@@ -431,3 +432,25 @@ class MessagingTemplateResponse(MessagingTemplateBase):
 class BulkPutMessagingTemplateResponse(BulkResponse):
     succeeded: List[MessagingTemplateResponse]
     failed: List[BulkUpdateFailed]
+
+
+class MessagingTemplateWithPropertiesBase(BaseModel):
+    id: Optional[str]  # Since summary returns db or defaults, this can be null
```

Comment:
Wait why can this be null?

Reply:
Items from the DB will have an id associated with it, but "defaults" in this context are the static strings we have set within Fides for each messaging type. 

Basically this is covers the case where not all messaging templates have been saved to the DB, for example, if an open source customer converts to paid, and has new access to the property-specific messaging Admin-UI page. 

This allows all templates to show up in the initial template list table.

Reply:
thank you for the explanation!